source/get-started/connect-to-mongodb.txt

```diff
@@ -0,0 +1,77 @@
+.. _railsmdb-get-started-connect-to-mongodb:
+
+==================
+Connect to MongoDB
+==================
+
+.. procedure::
+   :style: connected
+
+   .. step:: Create your Application
+
+      In your shell, run the following command to create a new application by
+      using {+cli-short+}:
+
+      .. code-block:: bash
+
+         railsmdb new quickstart
+
+      This command creates a new folder in your current directory called
+      ``quickstart`` and populates it with the scaffolding for a new application
+      that uses Mongoid with MongoDB.
+
+   .. step:: Assign the Connection String
+
+      In your shell, run the following command to navigate to the
+      ``config/mongoid.yml`` file that was automatically created in the preceding step:
+
+      .. code-block:: bash
+
+         cd config/mongoid.yml
+
+      Open the file in your editor of choice. Then, paste the 
```

Comment:
Does this listed `cd` command work work, given that `mongoid.yml` is not a directory? Intuitively I would think something like this might make more sense:

```suggestion
      In your shell, run the following command to navigate to the
      ``config`` directory that was automatically created in the preceding step:

      .. code-block:: bash

         cd config

      Within the ``config`` directory, open the ``mongoid.yml`` file in your editor of choice. Then, paste the 
```

Reply:
Good catch, thanks! You're right that wouldn't have worked, that was a bad translation on my part. Updated this to your suggestion